manifest.php

```diff
@@ -30,7 +31,7 @@
   'label'       => 'Delivery Management',
   'description' => 'Manages deliveries using the ontology',
   'license'     => 'GPL-2.0',
-  'version'     => '11.8.2',
+  'version'     => '11.9.0',
     'author'      => 'Open Assessment Technologies SA',
     'requires'    => [
         'generis'     => '>=12.15.0',
```

Comment:
Could you update generis to the right version pls?

Reply:
was waiting for other PR to be merged as I did not know what version I would end up with

Reply:
done